api/v1alpha1/instaslice_types.go

```diff
@@ -56,105 +33,161 @@ const (
 	AllocationStatusCreated  AllocationStatus = "created"
 )
 
-// Define the struct for allocation details
-type AllocationDetails struct {
-	// profile requested by user workload
-	// +required
-	Profile string `json:"profile"`
-	// start position of a profile on a gpu
-	// +required
-	Start int32 `json:"start"`
-	// size of profile that begins from start position
-	// +required
-	Size int32 `json:"size"`
-	// podUUID represents uuid of user workload
-	// +required
-	PodUUID string `json:"podUUID"`
-	// gpuUUID represents gpu uuid of selected gpu
-	// +required
+type MigSliceProfile string
+
+const (
+	Profile1G5GB   MigSliceProfile = "1g.5gb"
```

Comment:
I think we should remove constant or play a catchup game with different profiles that H100, GB200 or H800 supports

Reply:
That's a good point, I will remove it.